portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -640,13 +640,8 @@
 	<td></td>
 </tr>
 <tr>
-	<td>PARAGRAPH_SPECIFIC_FIELD_BY_BODY</td>
-	<td>//div[@class='liferay-ddm-form-field-paragraph-text']/p[contains(.,'${key_bodyName}')]</td>
-	<td></td>
-</tr>
-<tr>
-	<td>PARAGRAPH_LINK_SELECTOR</td>
```

Comment:
@leticiachiu, is the path `PARAGRAPH_LINK_SELECTOR` no longer necessary? Can you please give more details about it?

Reply:
no, this path is not used in any macro or testcase.. it was created before for [this test ](https://github.com/leticiachiu/liferay-portal/commit/f772a476c8dee37f8b2b99b02c109b78619bd5b4)but since the first test fix, it's not necessary anymore. i think someone forgot to remove it before.